web/pdf_thumbnail_viewer.js

```diff
@@ -135,15 +135,16 @@ var PDFThumbnailViewer = (function PDFThumbnailViewerClosure() {
       this.thumbnails = [];
       this._pagesRotation = 0;
       this._pagesRequests = [];
+
+      var container = this.container;
+      while (container.hasChildNodes()) {
```

Comment:
Similar to the above, I was trying to make this be the same as the existing code in `PDFViewer`, please see https://github.com/mozilla/pdf.js/blob/master/web/pdf_viewer.js#L419.
Do you want both of those files changed here?